code/modules/surgery/hepatectomy.dm

```diff
@@ -0,0 +1,48 @@
+/datum/surgery/hepatectomy
+	name = "Hepatectomy"
+	target_mobtypes = list(/mob/living/carbon/human, /mob/living/carbon/monkey)
+	possible_locs = list(BODY_ZONE_CHEST)
+	requires_real_bodypart = 1
+	steps = list(/datum/surgery_step/incise,
+		/datum/surgery_step/retract_skin,
+		/datum/surgery_step/saw,
+		/datum/surgery_step/clamp_bleeders,
+		/datum/surgery_step/incise,
+		/datum/surgery_step/hepatectomy,
+		/datum/surgery_step/close
+		)
+
+/datum/surgery/hepatectomy/can_start(mob/user, mob/living/carbon/target)
+	var/obj/item/organ/liver/L = target.getorganslot(ORGAN_SLOT_LIVER)
+	if(L.damage > 50 && !(L.organ_flags & ORGAN_FAILING))
+		return TRUE
+	else
+		return FALSE
+
+////hepatectomy, removes damaged parts of the liver so that the liver may regenerate properly
+//95% chance of success, not 100 because organs are delicate
+/datum/surgery_step/hepatectomy
+	name = "remove damaged liver section"
+	implements = list(/obj/item/scalpel = 95, /obj/item/melee/transforming/energy/sword = 65, /obj/item/kitchen/knife = 45,
+		/obj/item/shard = 35)
+	time = 52
+
+/datum/surgery_step/hepatectomy/preop(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
+	display_results(user, target, "<span class='notice'>You begin to cut out a damaged peice of [target]'s liver...</span>",
+		"<span class='notice'>[user] begins to make an incision in [target].</span>",
+		"<span class='notice'>[user] begins to make an incision in [target].</span>")
+
+/datum/surgery_step/hepatectomy/success(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
+	var/mob/living/carbon/human/H = target
+	H.setOrganLoss(ORGAN_SLOT_LIVER, 10) //not bad, not great
+	display_results(user, target, "<span class='notice'>You successfully remove the damaged part of [target]'s liver.</span>",
+		"<span class='notice'>[user] successfullly removes the damaged part of [target]'s liver.",
+		"<span class='notice'>[user] successfullly removes the damaged part of [target]'s liver.")
+	return TRUE
+
+/datum/surgery_step/hepatectomy/failure(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery)
+	var/mob/living/carbon/human/H = target
+	H.adjustOrganLoss(ORGAN_SLOT_LIVER, 15)
+	display_results(user, target, "<span class='warning'>You cut the wrong part of [target]'s liver!</span>",
+		"<span class='warning'>[user] cuts the wrong part of [target]'s liver!",
+		"<span class='warning'>[user] cuts the wrong part of [target]'s liver!")
```

Comment:
I think this reqs . = FALSE

Reply:
i don't exactly know what that would do. please forgive my pea brain

Reply:
well i referenced a bunch of other unique surgery steps and they don't seem to have . = FALSE and considering your uncertainty i'm just going to assume it's not really required

Reply:
don't resolve conversations if you didn't do anything please.

I just checked and it indeed does because otherwise you aren't prompted to know that False/True matters (if failure returns true it advances to the next step).

other surgeries should have this as well, they just get by with it now because null and false both fail the check.